tools/hlint.sh

```diff
@@ -22,9 +21,9 @@ while getopts ':f:m:k' opt
             ;;
          m) m=${OPTARG}
             if [ "$m" = "inplace" ]; then
-              check=false
+            :
```

Comment:
What does this do? Is this just a no-op?

Reply:
Yeah, that's the weird syntax for no-op in bash